test-backend/backend.env

```diff
@@ -0,0 +1,5 @@
+# Non-secret confiugration specific to the TPP backend
+BACKEND=expectations
```

Comment:
Does this want to be `test` now?

Reply:
Not yet - in job-runner speak, the magic word to get the dummy path is 'expectations' :)

Reply:
You were right, I was wrong! :facepalm: